### ISSUE TITLE
Single threaded?

### ISSUE THREAD

**Issue by Drefsab** [MattHodge/Graphite-PowerShell-Functions#42]
Hi there im wanting to monitor metrics from a fairly busy web app hosted on an IIS box. I came across this which is great but the script takes about 120 seconds on a big box (16 cpu 64gb ram etc) to push all the metrics I want to graphite. Im assuming this is because its processing them in single threaded fashion. Is there a way to make this multi-threaded at all?

**Comment by MattHodge** [comment]
Yeah, currently its single threaded - I agree it would be better if it sent all the metrics at once rather than one at a time.

**Comment by MattHodge** [comment]
I may have a solution to this... PowerShell Jobs may help out..

**Comment by wabit** [comment]
Do you have any update on this as I am also hitting a wall when processing a lot of metrics
